lib/oli_web/live/projects/create_project_modal.ex

```diff
@@ -36,10 +39,11 @@ defmodule OliWeb.Projects.CreateProjectModal do
               <div class="form-label-group">
                 <%= text_input(f, :title,
                   required: true,
+                  id: "project-title-placeholder",
                   class: "block min-w-full",
```

Comment:
We could get the same result with tailwind:

`class: "block min-w-full placeholder-[#9ca3af] dark:placeholder-[#eeebf5]/70"`

I think it is more "straightforward" and avoids us having to define an id and a specific class for that id in app.css. WDYT?

Reply:
Nice. I didn't know about this tailwind class. Thanks!